src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/Confirm.cshtml

```diff
@@ -10,27 +10,29 @@
         <div class="govuk-width-container">
             <div class="govuk-grid-row app-page-header__row">
                 <div class="govuk-grid-column-two-thirds">
-                    <h1 class="govuk-heading-xl app-page-header__heading">My apprenticeship</h1>
+                    <h1 class="govuk-heading-xl app-page-header__heading">My apprenticeship details</h1>
                 </div>
             </div>
         </div>
     </div>
 
 }
 
-<h2 class="govuk-heading-l">Confirm your apprenticeship</h2>
-<div class="govuk-warning-text">
-    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-    <strong class="govuk-warning-text__text">
-        <span class="govuk-warning-text__assistive">Warning</span>
-        You have 14 days to confirm your apprenticeship
-    </strong>
+<div asp-show="!@Model.ApprenticeshipConfirmed">
```

Comment:
There is also an `asp-hide` directive that (I think) clarifies readability as it doesn't involve a negative.

```suggestion
<div asp-hide="@Model.ApprenticeshipConfirmed">
```